CONTRIBUTING.md

```diff
@@ -219,6 +219,13 @@ Runtime code is held to the same standard as regular Envoy code, so both the old
 path and the new should have 100% coverage both with the feature defaulting true
 and false.
 
+Please note that if adding a runtime guarded feature, your [release notes](docs/root/version_history/current.rst) should include both the functional change, and how to revert it, for example
+
+```
```

Comment:
how about ` ```rst ` - i think it should work with pygments

Reply:
(conscious that other part of this file dont have codeblock hints - but in this case it would have clarified to me why there was double backticks in an md file - as well as making output pretty)

Reply:
if we do that, does it format the RST "correctly"?  I think we want the backticks visible so folks could copy paste, not so it looks like it would in a rst doc.

Reply:
it highlights it correctly as source - not renders it